tests/compilation_tests/robot_test/traverse_standard_test.py

```diff
@@ -47,6 +70,16 @@ def test3(self):
         allowed_dist_error = 0.5
         move_to_target_node(r2d2_state,
                             (43.444250, -76.483682), allowed_dist_error, database)
+    #Runs test3 using simple_move_to_target_node
+    def test3_simple(self):
+        r2d2_state = Robot_State(xpos=42.444250, ypos=-76.483682, heading=0,
+                                 epsilon=0.2, max_velocity=0.5, radius=0.2, phase=2, position_kp=.1, heading_kp=.1, position_kd=0.0, heading_kd=0.0
+                                 )
+        r2d2 = Robot(robot_state=r2d2_state)
+        database = DataBase(r2d2)
+        allowed_dist_error = 0.5
+        simple_move_to_target_node(r2d2_state,
+                            (43.444250, -76.483682), allowed_dist_error, database)
 
 # Takes approx. 9 seconds for 5 tests
```

Comment:
lets remove this as well

Reply:
What exactly should I remove? The entire test3? (did you highlight a specific part of the code? if you did, i don't think i see it)

Reply:
duplicate code, unnecessary args

Reply:
Duplicate code and unnecessary args removed via setUp method